src/components/WalletInfo/WalletInfo.scss

```diff
@@ -12,7 +12,7 @@
   }
 
   &--is-banner {
```

Comment:
We no longer need this class, all it was doing was setting a bgColor so if you don't need/want it anymore then remove the class.

Reply:
i guess this change will be require when the pr will be ready to merge (resolve conflict with develop branch).
For now, i still have the isBanner prop from the WalletInfo component, will fit it once the merge with develop done.

Reply:
@Goddak Yeah not a fan, can you change it maybe in your branch?